### ISSUE TITLE
The indicator given by Frank's algorithm

### ISSUE THREAD

**Issue by JerryChen97** [JerryChen97/KitaevLadder#41]
Following Frank Pollmann's [paper](https://arxiv.org/abs/1204.0704) as well as their code, I calculated the same quantity `O = tr(Ux Uy Ux^\dagger Uy^\dagger) / chi` for the Kitaev ladder at `J=(1.75, 0.25, 1.0)` and `J=(0.25, 1.75, 1.0)` and both gave a result `-1`.
According to their paper, such a result indicates that the phase point of interest is in a non-trivial phase (for example, Haldane phase in the spin-1 Haldane model).
I think up to now their codes still can't help us with the distinguishable properties between the large-x limit and the large-y limit, but indeed we can say almost affirmatively that both are SPT as our expectation.
After sorted everything I will update the Overleaf documentation with the most relevant figures; however, on the physical meaning of the quantity `O = tr(Ux Uy Ux^\dagger Uy^\dagger) / chi` I still need some time to read some literature about SPT.

**Comment by JerryChen97** [comment]
![image](https://user-images.githubusercontent.com/21050723/100152625-f82f5b00-2e70-11eb-9424-f70af970375d.png)

**Comment by JerryChen97** [comment]
Hi @aaronszasz , this is a fig given by reproducing the algorithm mentioned in Frank's paper in TeNPy3, formula (14) in [arxiv 1204.0704](https://arxiv.org/pdf/1204.0704.pdf).
The point `Jx=1.5` is the gapless point so I think it should be fine to be away from the supposed value; still the `-1` values in the SPT side are sometimes varying around 1 and I am not sure whether or not this is good enough.

**Comment by JerryChen97** [comment]
In their paper, the results seem to be super precise, very very close to `-1` in SPT phase, and those values are almost aligned on a horizontal line.

**Comment by aaronszasz** [comment]
I'm actually not sure how precise the results ought to be, but I definitely see your point that the results in Frank's paper are much more precise.  My guess is that this is already good enough, but I'm not immediately sure.

**Comment by JerryChen97** [comment]
Yeah I think qualitatively this fig is good enough since it properly indicates both the SSB phase and SPT phase.
I am trying some other parameters to see if I can improve it a little bit

**Comment by JerryChen97** [comment]
Maybe a more important issue will be that this method still can not help distinguish between large X and large Y; at both sides, the detector just tells us the same `-1`.
I am actually wondering about the possibility of distinguishing them: from the perspective of SPT, are they really much different? From Frank's paper, I got a feeling that for a phase where D2 symmetry is preserved, the information they can provide is whether the phase is trivial (site factorizable; the indicator shows `+1`) or non-trivial (SPT; the indicator shows `-1`).

**Comment by JerryChen97** [comment]
![image](https://user-images.githubusercontent.com/21050723/100156997-de454680-2e77-11eb-8c04-fad65017c3fe.png)
A higher-resolution figure.
It seems like something weird happened around `Jx=1.8`...

**Comment by aaronszasz** [comment]
Well, that's a bit strange, but in this figure with higher resolution I think it's even more clear that you've successfully demonstrated the SPT nature of the phase.  Do the errors here correlate at all with errors in expectation values etc?  It could be that the wavefunctions are just less accurate for those points.

**Comment by aaronszasz** [comment]
About the X versus Y phases, I would have thought that the symmetry operators you need to use are different.
